src/client/opamClient.ml

```diff
@@ -330,9 +333,11 @@ let upgrade_t
                  ) (OpamPackage.Set.elements unopt))
            );
          OpamConsole.formatted_msg
-           "However, you may \"opam upgrade\" these packages explicitly, \
+           "However, you may \"opam upgrade\" these packages explicitly \
+            at these versions (e.g. \"opam upgrade %s\"), \
```

Comment:
```suggestion
            at these versions (e.g. 'opam upgrade %s'), \
```
\+ some underline ?

Reply:
for underline, i'm not sure if it's necessary. The rest of the opam example commands in our codebase uses either double-quotes like here or `bold`, the former being used more often.

Reply:
They are both used in fact, `"` and `'`.